grammars/erlang.bnf

```diff
@@ -321,6 +324,32 @@ argument_definition_list ::= '(' argument_definition? (',' argument_definition)*
 clause_guard ::= when <<guarded guard>> {pin=1}
 clause_body ::= '->' exprs {pin=1}
 
+app_expression ::= empty '{' 'application' ',' module_ref ',' app_parameters '}' {elementType=tuple_expression}
```

Comment:
Remove empty, and add this dummy rule instead. At least, we'll know it's some hack we're doing, instead of adding confusing matching and pinning to our rules.
```
private is_app_hack ::= is_app 'xxx' //TODO remove when Grammar-Kit won't generate a lookahead for '{' in is_app code
```